docs/bosh/README.md

```diff
@@ -81,40 +111,42 @@ The following instructions offer the fastest path to getting BOSH up and running
 
 1. Clone this repository and go into the BOSH docs directory:
 
-  ```
-  git clone https://github.com/cloudfoundry-incubator/bosh-google-cpi-release.git
-  cd bosh-google-cpi-release/docs/bosh
-  ```
+   ```
+   git clone https://github.com/cloudfoundry-incubator/bosh-google-cpi-release.git
+   cd bosh-google-cpi-release/docs/bosh
+   ```
 
 1. View the Terraform execution plan to see the resources that will be created:
 
-  ```
-  docker run -i -t \
-    -e "GOOGLE_CREDENTIALS=${GOOGLE_CREDENTIALS}" \
-    -v `pwd`:/$(basename `pwd`) \
-    -w /$(basename `pwd`) \
-    hashicorp/terraform:light plan \
-      -var service_account_email=${service_account_email} \
-      -var projectid=${project_id} \
-      -var region=${region} \
-      -var zone=${zone} \
-      -var baseip=${base_ip}
-  ```
+   ```
+   docker run -i -t \
+     -e "GOOGLE_CREDENTIALS=${GOOGLE_CREDENTIALS}" \
+     -v `pwd`:/$(basename `pwd`) \
+     -w /$(basename `pwd`) \
+     hashicorp/terraform:light plan \
+       -var service_account_email=${service_account_email} \
+       -var projectid=${project_id} \
+       -var region=${region} \
+       -var zone=${zone} \
+       -var baseip=${base_ip} \
+       -var xpn_host_projectid=${xpn_host_project_id-project_id}
+   ```
 
 1. Create the resources (should take between 60-90 seconds):
 
-  ```
-  docker run -i -t \
-    -e "GOOGLE_CREDENTIALS=${GOOGLE_CREDENTIALS}" \
-    -v `pwd`:/$(basename `pwd`) \
-    -w /$(basename `pwd`) \
-    hashicorp/terraform:light apply \
-      -var service_account_email=${service_account_email} \
-      -var projectid=${project_id} \
-      -var region=${region} \
-      -var zone=${zone} \
-      -var baseip=${base_ip}
-  ```
+   ```
+   docker run -i -t \
+     -e "GOOGLE_CREDENTIALS=${GOOGLE_CREDENTIALS}" \
+     -v `pwd`:/$(basename `pwd`) \
+     -w /$(basename `pwd`) \
+     hashicorp/terraform:light apply \
+       -var service_account_email=${service_account_email} \
+       -var projectid=${project_id} \
+       -var region=${region} \
+       -var zone=${zone} \
+       -var baseip=${base_ip} \
+       -var xpn_host_projectid=${xpn_host_project_id-project_id}
```

Comment:
This is intentional. At this stage if the user hasn't set `xpn_host_project_id` we will default to `project_id` meaning all network resources will be created in the same project.